player_test.py

```diff
@@ -174,62 +175,66 @@ def test_returns_user_input_for_input_is_red(self):
 
   def test_returns_user_input_for_input_is_yellow(self):
   # test 39
-    mario = Player("placeholder name")
+    mario = Player("mario")
     allowed_choices = ['red', 'yellow', 'green', 'purple', 'brown']
+    prompt_text = "please enter your choice of ant"
     expected_result = "yellow"
     input_patcher = mock.patch('builtins.input', return_value = "yellow")
     InputMock = input_patcher.start()
```

Comment:
`input_mock` see above. I won't comment in all of them, so if you choose to change this naming, then make sure you change it in all places.